src/OrbitQt/MoveFilesDialog.ui

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file doesn't belong in this change, does it?

Reply:
ouch... right sorry. removing.